### PR TITLE
Linked cohorts showcase

### PR DESCRIPTION
**Pull request by z-w123**
Hi,
this PR is for the sandbox version of the linked cohort data showcase page.

thanks in advance,
Zahra

### CODE REVIEW

File: showcase/linked-cohort-data.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Linked Cohort Data
+title: Linked pathogen and host data across archives - a multi-omics, SARS-CoV-2 cohort case study
 contributors: [Gabriele Rinck, Zahra Waheed] 
-description: Here we describe the first example of linking and sharing a cohort data set via the EMBL-EBI infrastructure. 
-affiliations: [ELIXIR Europe]
+description: Here we describe the first example of linking and sharing a COVID-19, multi-omics cohort data set via the EMBL-EBI infrastructure. 
+affiliations: [EMBL-EBI]
```
Comment (bedroesb) [inline showcase/linked-cohort-data.md:5]: don't forget to add information about this affiliation to the _data/affiliations.yml file!

File: showcase/linked-cohort-data.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
-title: Linked Cohort Data
+title: Linked pathogen and host data across archives - a multi-omics, SARS-CoV-2 cohort case study
 contributors: [Gabriele Rinck, Zahra Waheed]
```
Comment (bedroesb) [inline showcase/linked-cohort-data.md:3]: Same goes for these contributors in the _data/CONTRIBUTORS.yml file :)

**Comment by bedroesb** [comment]
@z-w123 thanks for this! Hopefully we can soon create a PR from you branch to the original repo after we approve the try-out on this repo! See my small comments to finalize

File: showcase/linked-cohort-data.md (hunk 1 above)
Comment (z-w123) [reply showcase/linked-cohort-data.md:5]: Ah thanks, done!

**Comment by z-w123** [comment]
Thanks @bedroesb! Have addressed the comments

**Review by bedroesb — APPROVED** [review]

**Comment by bedroesb** [comment]
lets have a look how they look like live!

**Pull request merged**